tokens/src/core/components/Carousel.json

```diff
@@ -29,38 +29,33 @@
     "carousel": {
       "base": {
         "source": "$carousel-transition",
-        "$value": [
-          {
-            "property": "opacity",
-            "duration": "0ms",
-            "delay": "0ms",
-            "timingFunction": "ease"
-          },
-          {
-            "property": "transform",
-            "duration": "{transition.carousel.duration}",
-            "delay": "0ms",
-            "timingFunction": "ease-in-out"
-          }
-        ]
+        "$value": {
+          "property": "transform",
+          "duration": "{transition.carousel.duration}",
+          "timingFunction": "ease-in-out",
+          "delay": "0ms",
+          "behavior": "normal"
+        }
```

Comment:
[inform]: According to the original token, the `$carousel-transition` has one value:
```
"base": { "source": "$carousel-transition", "$value": "transform {transition.carousel.duration} ease-in-out", "$type": "transition" },
```